net/nebula/Makefile

```diff
@@ -58,64 +59,59 @@ define Package/nebula-proto
 endef
 
 define Package/nebula-service
-  SECTION:=net
-  CATEGORY:=Network
+  $(call Package/nebula/Default)
   TITLE:=nebula-service
-  URL:=https://docs.openwrt.melmac.net/nebula/
   DEPENDS:=nebula
 	DEPENDS+=+!BUSYBOX_DEFAULT_AWK:gawk
 	DEPENDS+=+!BUSYBOX_DEFAULT_SED:sed
 	CONFLICTS:=nebula-proto
   PKGARCH:=all
 endef
 
-define Build/Compile
-  $(call GoPackage/Build/Compile)
+define Package/nebula/description/Default
+  Nebula is a scalable overlay networking tool with a focus on performance, simplicity
+  and security. It lets you seamlessly connect computers anywhere in the world.
 endef
 
 define Package/nebula/description
-Nebula is a scalable overlay networking tool with a focus on performance, simplicity
-and security. It lets you seamlessly connect computers anywhere in the world.
-This package contains only nebula binary. Unless you want to start nebula manually,
-you may want to also install *either* 'nebula-service' *or* 'nebula-proto' package.
+  $(call Package/nebula/description/Default)
+  This package contains only nebula binary. Unless you want to start nebula manually,
+  you may want to also install *either* 'nebula-service' *or* 'nebula-proto' package.
 endef
 
 define Package/nebula-cert/description
-Nebula is a scalable overlay networking tool with a focus on performance, simplicity
-and security. It lets you seamlessly connect computers anywhere in the world.
-This package contains only nebula-cert binary.
+  $(call Package/nebula/description/Default)
+  This package contains only nebula-cert binary.
 endef
 
 define Package/nebula-proto/description
-Nebula is a scalable overlay networking tool with a focus on performance, simplicity
-and security. It lets you seamlessly connect computers anywhere in the world.
-This package contains only OpenWrt protocol/interface support for nebula.
+  $(call Package/nebula/description/Default)
+  This package contains only OpenWrt protocol/interface support for nebula.
 endef
 
 define Package/nebula-service/description
-Nebula is a scalable overlay networking tool with a focus on performance, simplicity
-and security. It lets you seamlessly connect computers anywhere in the world.
-This package contains only OpenWrt-specific init.d script for nebula.
+  $(call Package/nebula/description/Default)
+  This package contains only OpenWrt-specific init.d script for nebula.
+endef
+
+define Package/nebula/conffiles
+/etc/nebula/
 endef
 
+Package/nebula-cert/conffiles = $(Package/nebula/conffiles)
+Package/nebula-proto/conffiles = $(Package/nebula/conffiles)
+Package/nebula-service/conffiles = $(Package/nebula/conffiles)
```

Comment:
Are these really necessary? At least `nebula-proto` and `nebula-service` depend on `nebula` which has conffiles defined already.

Reply:
Good catch, thanks!